doc/manuals/contribution_guide.md

```diff
@@ -0,0 +1,674 @@
+# Orion Context Broker Contribution Guide
+
+This document describes the guidelines to contribute to the Orion Context Broker open source repository. If you are
```

Comment:
to contribute to Orion Context Broker . I think open source repository is not needed